src/Facility.ConformanceApi/Answer.g.cs

```diff
@@ -45,7 +63,7 @@ public partial struct Answer : IEquatable<Answer>
 		/// <summary>
 		/// Checks for equality.
 		/// </summary>
-		public bool Equals(Answer other) => StringComparer.OrdinalIgnoreCase.Equals(ToString(), other.ToString());
+		public bool Equals(Answer other) => StringComparer.Ordinal.Equals(ToString(), other.ToString());
```

Comment:
Non-normalized values should still be compared case-insensitively, so these should be reverted.